crates/proof-of-sql/src/base/database/owned_and_arrow_conversions.rs

```diff
@@ -193,63 +193,47 @@ impl<S: Scalar> TryFrom<&ArrayRef> for OwnedColumn<S> {
                     let array = value
                         .as_any()
                         .downcast_ref::<TimestampSecondArray>()
-                        .ok_or_else(|| {
-                            OwnedArrowConversionError::UnsupportedTimestampUnit(
-                                "Second".to_string(),
-                            )
-                        })?;
+                        .ok_or_else(|| InvalidTimeUnit("Second".to_string()))?;
```

Comment:
Hmm why `InvalidTimeUnit("Second".to_string()))`? Second is definitely a valid time unit.